draft-pda-protocol.md

```diff
@@ -670,17 +672,51 @@ collect request is said to be valid if all of the following conditions hold (let
 The leader responds to valid collect requests by first interacting with the
 helper as described in {{pa-aggregate}}. Once it has obtained the helper's
 encrypted output share for the batch, it responds to the collector's request
-with the following message:
+with HTTP status 200 and a body consisting of the following message:
 
 ~~~
 struct {
-  PPMTaskID task_id;
-  PPMProto proto;
-  PPMOutputShare leader_share;
-  opaque encrypted_helper_share;
+  PPMEncryptedOutputShare encrypted_leader_share;
+  PPMEncryptedOutputShare encrypted_helper_share;
 } PPMCollectResp;
+
+struct {
+  uint8 collector_hpke_config_id;
+  opaque enc<1..2^16-1>;
+  opaque encrypted_output_share<1..2^16>;
+} PPMEncryptedOutputShare;
+~~~
+
+* `collector_hpke_config_id` is `collector_config.id` from the `PPMParam`
+  corresponding to `PPMCollectReq.task_id`.
+* `enc` is the encapsulated HPKE context, used by collector to decrypt the
+  output share.
+* `encrypted_output_share` is an encrypted `PPMOutputShare`, whose structure is
+  given below.
+
+~~~
+struct {
+  select (PPMParam.proto) { // PPMParam for the PPM task
+    case prio: PrioOutputShare;
+    case hits: HitsOutputShare;
+  }
+} PPMOutputShare;
 ~~~
 
+A `PPMOutputShare` is encrypted as follows:
+
+~~~
+enc, context = SetupBaseS(pk, "pda output share" || server_role)
```

Comment:
Besides moving this definition to here from the "Aggregate Request" section, I also appended `server_role` to the `info` string passed to `SetupBaseS`.